gauge_web_app_steps/web_app_steps.py

```diff
@@ -737,16 +737,16 @@ def assert_url_contains(expected_url_param: str) -> None:
         _err_msg(f"url {current_url} does not contain {expected_url}")
 
 
-@step("Assert <by> = <by_value> is displayed")
-def assert_element_is_displayed(by: str, by_value: str) -> None:
+@step("Assert <by> = <by_value> exists")
```

Comment:
We have to keep the old step as an alias.
https://docs.gauge.org/writing-specifications.html?os=macos&language=python&ide=vscode#step-alias

```suggestion
@step(["Assert <by> = <by_value> exists", "Assert <by> = <by_value> is displayed"])
```

Then you need to check which alias was used. Maybe you can find this info in 
`data_store.spec[app_context_key].step.text`. I am not sure. Depends if the context class is updated between every step. Otherwise this doc offers another approach: https://docs.gauge.org/writing-specifications.html?os=windows&language=python&ide=vscode#current-execution-context-in-the-hook
If the old alias was used, a warning should be logged.